modules/pokemon.py

```diff
@@ -915,6 +915,33 @@ def _decrypted_data(self) -> bytes:
         )
         return self.data[:32] + decrypted.tobytes() + self.data[80:100]
 
+    @cached_property
+    def _decrypted_data_export(self) -> bytes:
```

Comment:
Two things:

1. If you're using this method externally, i.e. from outside this class, it should not be prefixed with a `_` (which would mark it as a 'private' function by convention.)
2. I don't think this property needs to be cached, we're very unlikely to call it several times in a row on the same instance of `Pokemon`.

We already have a method that is used for some sort of export, `to_dict()`. Maybe instead of making this a method, we can call it `to_pk3()`?